src/snowflake/cli/api/project/schemas/native_app/path_mapping.py

```diff
@@ -6,7 +6,7 @@
 from snowflake.cli.api.project.schemas.updatable_model import UpdatableModel
 
 
-class AnnotationProcessor(UpdatableModel):
+class Processor(UpdatableModel):
```

Comment:
Updated name as `AnnotationProcessor` is too specific, a processor does not have to act only on the annotations.